Fix incorrect case of ldap sync status string

JonathonReinhart opened the pull request:
Here's a quick bug-fix.

Fixes #9562

The case of the LDAP sync status string was incorrect, causing all rows to use class "danger" and appear pink.

With this fix, successful imports now use class "success" and show up green, while failed import rows use "danger":

```html
                    <tr class="success">
              <td>jsmith</td>
              <td></td>
              <td>John</td>
              <td>Smith</td>
              <td>jsmith@example.com</td>
              <td>
                                  <i class="fa fa-check"></i> updated
                
                </td>
              </tr>
                    <tr class="danger">
              <td>jdoe</td>
              <td></td>
              <td>Jane</td>
              <td>Doe</td>
              <td>jdoe@example.com</td>
              <td>
                                  <span class="alert-msg" aria-hidden="true">The selected manager id is invalid.</span>
                
                </td>
              </tr>
```

JonathonReinhart committed:
Fix incorrect case of ldap sync status string

Fixes #9562

snipe commented:
Hi there - thanks for this! Can you re-apply this change to a fresh pull of the `develop` branch, per [our Contributing documentation](https://snipe-it.readme.io/docs/contributing-overview)? If I try to switch the target as it is now, it shows 43 files changed. :-/

Thanks!

JonathonReinhart commented:
Hi @snipe. I rebased it onto `develop` and changed the target branch accordingly.

Sorry about that. Most projects I work with have the [default branch](https://docs.github.com/en/github/administering-a-repository/changing-the-default-branch) set to the one that they expect PRs into, so I forgot that `snipe-it` wanted PRs to target `develop`.

snipe commented:
Hi @JonathonReinhart - no worries at all. I know we're a little weird, but we move a lot faster than most projects, and almost never commit directly to master. I so appreciate you taking the time. This is a solid fix, and I appreciate it. <3

snipe commented:
![rdj_thanks](https://user-images.githubusercontent.com/197404/118068024-7cf62e80-b356-11eb-9629-af803b3f70c1.gif)
